spec/requests/alert_definitions_spec.rb

```diff
@@ -306,4 +306,77 @@
     expect(alert_definition_profiles.first.reload.description).to eq("Updated Test Alert Profile 1")
     expect(alert_definition_profiles.second.reload.description).to eq("Updated Test Alert Profile 2")
   end
+
+  it "gets assignments for an alert definition profile" do
+    api_basic_authorize action_identifier(:alert_definition_profiles, :read, :resource_actions, :get)
+    alert_definition_profile = FactoryGirl.create(:miq_alert_set)
+    enterprise = FactoryGirl.create(:miq_enterprise)
+    enterprise_href = api_enterprise_url(nil, enterprise)
+    alert_definition_profile.assign_to_objects(enterprise)
+
+    get(api_alert_definition_profile_url(nil, alert_definition_profile))
+
+    expect(response).to have_http_status(:ok)
+    expect(response.parsed_body["get_assigned_tos"]["objects"]).to include(a_hash_including("href"=>enterprise_href))
+  end
+
+  it "assigns an alert definition profile to object" do
+    api_basic_authorize :alert_profile_assign
+    alert_definition_profile = FactoryGirl.create(:miq_alert_set)
+    enterprise = FactoryGirl.create(:miq_enterprise)
+    enterprise_href = api_enterprise_url(nil, enterprise)
+
```

Comment:
please add an expectation for the empty assignments before the assigning request

Reply:
Done.